README.md

```diff
@@ -32,9 +32,9 @@ File Guide (The Source):
 
 ## References for documentation team
 
-* We used AsciiDoc to document Stonesoup (https://docs.asciidoctor.org/asciidoc/latest/#about-asciidoc).
+* We used AsciiDoc to document App Studio (https://docs.asciidoctor.org/asciidoc/latest/#about-asciidoc).
 * We created this website using Antora (https://docs.antora.org/antora/latest/). We use Antora because it provides a search capability that we can customize.
-* We manage documentation in GitLab. They are in a repository (<https://gitlab.cee.redhat.com/red-hat-hybrid-application-cloud-build-services-documentation/Stonesoup-documentation>). We use GitLab to manage publication - continuous integration settings in the repository automatically generate the web site when changes are merged and GitLab Pages is used to publish the website here <https://red-hat-stone-soup.pages.redhat.com/stonesoup-documentation/>.
+* We manage documentation in GitLab. They are in a repository (<https://gitlab.cee.redhat.com/red-hat-trusted-application-pipeline/red-hat-trusted-application-pipeline>). We use GitLab to manage publication - continuous integration settings in the repository automatically generate the web site when changes are merged and GitLab Pages is used to publish the website here <https://red-hat-stone-soup.pages.redhat.com/stonesoup-documentation/>.
```

Comment:
I think we can update line # 21 line to convey info about the GitHub repo instead of GitLab. We no longer maintain the GitLab repo.